documentation/Logging-behaviour.md

```diff
@@ -0,0 +1,44 @@
+## Logging behaviour
+MSBuild has a few different built-in loggers, which have different behaviours depending on verbosity. For more information on loggers you can visit the [Microsoft Learn page](ttps://learn.microsoft.com/visualstudio/msbuild/obtaining-build-logs-with-msbuild), or take a look at the [high-level overview of MSBuild](https://github.com/dotnet/msbuild/blob/main/documentation/High-level-overview.md#diagnosability--loggers).
+
+If you are expecting to see a certain type of message (like test logs) but are unable to find it in one of our loggers, check if the verbosity is correct and if the message has the correct type and importance.
+
+### Message types
+There are various types of messages within MSBuild with different importances and purposes. .
+There are some message types that are built-in within the engine, `errors`, `warnings`, and MSBuild engine information. The custom messages, that can come either from the engine or other sources, are selected and displayed based on the `importance` of the message. There can be high, normal, and low importance messages being displayed.
```

Comment:
The displaying based on importance applies only for Console logger - so maybe that part of the information should be moved to that section?

Reply:
Oh I see you have nice tables for that below - so maybe just mention that here (something like 'detailed overview of what gets displayed is listed per individual loggers in matching sections')